hack/benchmark-go.sh

```diff
@@ -14,6 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# This script runs `make test` command with some args for benchmark test.
+# The "true" target of this makerule is `hack/make-rules/test.sh`.
+# Usage: `hack/benchmark-go.sh`.
```

Comment:
I would call out that this uses the WHAT env var and provide an additional example